src/main/content.ts

```diff
@@ -13,7 +13,7 @@ import { IpcEvents } from '../ipc-events';
 // parent directory of all the downloaded template fiddles
 const TEMPLATES_DIR = path.join(app.getPath('userData'), 'Templates');
 
-// location of the fallback template fiddle used iff downloading failed
+// location of the fallback template fiddle used if downloading failed
```

Comment:
I think this was intentional ([iff = if and only if](https://en.wikipedia.org/wiki/If_and_only_if))